packages/transaction-pay-controller/CHANGELOG.md

```diff
@@ -7,10 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.6.0]
```

Comment:
Nit: Are there any new features/enhancements in this version? Usually we use patch bumps for releases that just contain dependency updates